defaults/main.yml

```diff
@@ -9,3 +9,4 @@ appuio_openshift_acme_docker_image: quay.io/tnozicka/openshift-acme
 appuio_openshift_acme_docker_image_tag: controller-0.9
 appuio_openshift_acme_exposer_image: quay.io/tnozicka/openshift-acme
 appuio_openshift_acme_exposer_image_tag: exposer-0.9
+appuio_openshift_acme_default_rsa_key_bit_size: "4096"
```

Comment:
Can this be a number? We'll have to make sure to cast it to a string before using it in the template anyway